vantiq-cloud-infra-operations/terraform_azure/new/modules/opnode/init-script.sh

```diff
@@ -55,7 +55,18 @@ wget https://github.com/bitnami-labs/sealed-secrets/releases/download/v0.9.5/kub
 install -m 755 kubeseal /usr/local/bin/kubeseal
 
 # install az-cli
-curl -sL https://aka.ms/InstallAzureCLIDeb | bash
```

Comment:
This installation might not work.
- https://learn.microsoft.com/en-us/cli/azure/install-azure-cli-linux?pivots=apt#option-2-step-by-step-installation-instructions